go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aptible/terraform-provider-aptible-iaas
 go 1.18
 
 require (
-	github.com/aptible/cloud-api-clients/clients/go v0.0.0-20220907201222-ccf7b001ed80
+	github.com/aptible/cloud-api-clients/clients/go v0.0.0-20221012150224-d9a9a11c6363
```

Comment:
you should probably just pin this with the releases instead of latest, unless you want to use bleeding edge